packages/semcom-node/config/config-default.json

```diff
@@ -12,5 +13,28 @@
     "files-dasn:config/presets/decentralization.json"
   ],
   "@graph": [
+    {
+      "@id": "urn:semcom-node:default:MemoryStore",
+      "@type": "MemoryStore",
+      "initialData": ["peers", []],
+      "initialData": ["storage", ["http://localhost:9000/metadata/"]]
```

Comment:
If I understand correctly; this config is used to start the repository in production. If so; wouldn't this break something? Because I presume there's no pod server available in production on this uri.

Reply:
Hm, the point is there should be a pod server deployed with it.

Reply:
(or in any case, there should somewhere be a pod server to which this points, where the metadata is stored)

Reply:
Could we use an in-memory store as fallback?

Reply:
There was an inmemory config, but it does not yet work with the sync part.

Reply:
Might be a justifiable trade-off in this specific case?

Reply:
The problem is that, since it does not include the sync handlers, the launch code fails when those are not found.

Reply:
Maybe we could solve that with a boolean config variable?

Reply:
Would'nt it be better just to deploy a pod server with it? That is also the only way it's really useable (with the ability to upload new components with oidc auth etc.)